survey.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey</title>
+</head>
+
+<body>
+    <nav>
+        <ul id="top">
+            <li><a href="home.html">Home</a></li>
```

Comment:
Suggest to add the relative path to each of the html pages. I believe the relative path is not required. But I read that putting the relative path is a typical practice.